image/v3/modifying/text-fonts.md

```diff
@@ -110,7 +110,7 @@ base point. Possible values are left, right and center. Default: `left`
 
 ### Vertical alignment
 
-> public FontInterface::valign(string $valign): FontInterface
+> public FontInterface::setValignment(string $valign): FontInterface
```

Comment:
Please revert. This is correct call of class Intervention\Image\Typography\FontFactory::class.